src/components/app/details/appDetails/utils.tsx

```diff
@@ -204,6 +204,60 @@ export function ThroughputSelect(props) {
     />
 }
 
+export function LatencySelect(props) {
+    return <CreatableSelect className=""
+        placeholder="Latency"
+        value={{ label: props.latency, value: props.latency }}
+        options={[
+            { label: '.999', value: '.999' },
+            { label: '.995', value: '.995' },
+            { label: '.99', value: '.99' },
+            { label: '.95', value: '.95' }
+        ]}
+        onChange={props.handleLatencyChange}
+        styles={{
```

Comment:
[Refactoring] style can be extracted out to local variable (as same style is being used in throughput select).
However can be done later.